Components/ListIcons/functions.php

```diff
@@ -362,7 +362,17 @@ function createFeatherIconSelectChoices($iconKeys = [])
                 'type' => 'group',
                 'layout' => 'row',
                 'sub_fields' => [
-                    Api::loadFields('FieldVariables', 'theme')
+                    Api::loadFields('FieldVariables', 'theme'),
+                    [
+                        'label' => 'Item Alignment',
+                        'name' => 'align',
+                        'type' => 'button_group',
+                        'choices' => [
+                            'left' => 'Left',
```

Comment:
What do you think about using an icon like in BlockImageText?
https://github.com/flyntwp/flynt/blob/master/Components/BlockImageText/functions.php#L24